wasm_runtime/src/config.rs

```diff
@@ -210,7 +210,42 @@ impl WasmConfig {
         wasm_config.wasi_mapdirs.push((wasi_map.to_string(), wasi_dir.to_string()));
         Ok(())
     }
+    // Returns a version of path with all slashes converted to forward
+    fn path_to_unix_slashes(path: &Path) -> String {
```

Comment:
I believe can use `PathBuf` directly here.

Reply:
Also, maybe this function is not needed:

In `find_longest_map` just invoke `replace()` in the String and then convert it into PathBuf?

Reply:
Based on the code, I see we're doing several conversions between these two types:

```
&str -> Path -> &str -> String
```

I think we can simplify this by making this method to work with strings and then, changing the other calls to use the right types:

```
fn path_to_unix_slashes(path: &str) -> String {
  path.to_string().replace("\\", "/")
}

// Then...

let cleaned_path = clean(Self::path_to_unix_slashes(path)); 

// When computing the final path, we can rely on the unix separator directly
Some(format!("{to}/{tail}"))

// And finally
longest_map.map(|p| clean(Self::path_to_unix_slashes(&p)))
```

Wdyt?